teste_stateless_statefull/lib/view/statefulView.dart

```diff
@@ -43,7 +43,6 @@ class _StatefulViewState extends State<StatefulView> {
                 setState(() {
                   _numeroRandomicoResultado = obtemNumeroRandomico();
```

Comment:
Esta linha está errada
```suggestion
                  _numeroRandomicoResultado = obtemNumeroRandomico(); =D
```